:w

```diff
@@ -0,0 +1,139 @@
+    created_at                      { creation_time }
+    creator                         { create(:user, :orphan) }
+  end
+
+  trait :third_party do
+    third_party { true }
+    third_party_company_name { 'Foogle and Sons Solicitors at Law' }
+    third_party_name { 'Mr J. Smith' }
+    third_party_reference { 'FOOG1234' }
+    third_party_relationship { 'Solicitor' }
+  end
+
+  trait :data_to_be_requested do
+    # Default state for a new offender_sar_case
+  end
+
+  trait :waiting_for_data do
+    transient do
+      identifier { 'Waiting for data Offender SAR' }
+    end
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      kase.reload
+    end
+  end
+
+  trait :ready_for_vetting do
+    transient do
+      identifier { 'Ready for vetting Offender SAR' }
+    end
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      create :case_transition_ready_for_vetting, case: kase
+      kase.reload
+    end
+  end
+
+  trait :vetting_in_progress do
+    transient do
+      identifier { 'Vetting in progress Offender SAR' }
+    end
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      create :case_transition_ready_for_vetting, case: kase
+      create :case_transition_vetting_in_progress, case: kase
+      kase.reload
+    end
+  end
+
+  trait :ready_to_copy do
+    transient do
+      identifier { 'Ready to close Offender SAR' }
+    end
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      create :case_transition_ready_for_vetting, case: kase
+      create :case_transition_vetting_in_progress, case: kase
+      create :case_transition_ready_to_copy, case: kase
+      kase.reload
+    end
+  end
+
+  trait :ready_to_dispatch do
+    date_responded { Date.today }
+    info_held_status { find_or_create :info_status, :held }
+    transient do
+      identifier { 'Ready to dispatch Offender SAR' }
+    end
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      create :case_transition_ready_for_vetting, case: kase
+      create :case_transition_vetting_in_progress, case: kase
+      create :case_transition_ready_to_copy, case: kase
+      create :case_transition_ready_to_dispatch, case: kase
+      kase.reload
+    end
+  end
+
+  trait :closed do
+    transient do
+      identifier { 'Closed Offender SAR' }
+    end
+
+    received_date  { 22.business_days.ago }
+    date_responded { 4.business_days.ago }
+
+    after(:create) do |kase|
+      create :case_transition_waiting_for_data, case: kase
+      create :case_transition_ready_for_vetting, case: kase
+      create :case_transition_vetting_in_progress, case: kase
+      create :case_transition_ready_to_copy, case: kase
+      create :case_transition_ready_to_dispatch, case: kase
+      create :case_transition_close, case: kase
+      kase.reload
+    end
+  end
+end
```

Comment:
I think you might have created this file accidentally on the way past... `:w` is a very vim-sounding filename :)

Reply:
lol thanks vim